src/app.service.ts

```diff
@@ -3,6 +3,12 @@ import { Injectable } from '@nestjs/common';
 @Injectable()
 export class AppService {
   getHello(): string {
-    return 'Hello World!';
+    return 'Hello World how is it looking right now!';
   }
-}
+
+  get(): string {
+    console.log('hello world, how is ig!')
+    const maths = 1 +2+5;
+    return `${maths} Hello Prima. how are youdoing what is the total of given number and 20%`;
+  }
```

Comment:
PullProofAI

The `get()` method in `AppService` seems to have unnecessary console logs and unclear return statements. It's recommended to clean up the method for better readability and maintainability.

```diff
-     console.log('hello world, how is ig!')
-     const maths = 1 +2+5;
-     return `${maths} Hello Prima. how are youdoing what is the total of given number and 20%`;
+     const total = 1 + 2 + 5;
+     return `Total: ${total}. Hello Prima, how are you doing? What is the sum of the given numbers and 20%?`;
```

### Comments:

- The `get()` method should have clear and concise logic without unnecessary console logs.
- Improved variable naming and return statement for better clarity.


<!-- This is an auto-generated comment by PullProofAI -->